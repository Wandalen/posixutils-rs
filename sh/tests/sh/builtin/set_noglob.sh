cd $TEST_READ_DIR

set -f

echo *.txt

set +f

echo *.txt