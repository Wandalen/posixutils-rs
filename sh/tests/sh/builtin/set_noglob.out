*.txt
file1.txt file2.txt filea.txt fileb.txt filec.txt
